### PR TITLE
Add filter invitation 2

### PR DESCRIPTION
**Pull request by jeremie74**

### CODE REVIEW

File: libs/utils/src/lib/static-model/static-model.ts [hunk 1]
```diff
@@ -425,8 +425,20 @@ export const orgActivity = {
 
 /** Status of an Organization, set to pending by default when an Organization is created. */
 export const organizationStatus = {
-  pending: 'Pending',
-  accepted: 'Accepted'
+  accepted: 'Accepted',
+  declined: 'Declined',
```
Comment (phpgeekfr) [inline libs/utils/src/lib/static-model/static-model.ts:429]: why do you update this const ?

Comment (jeremie74) [reply libs/utils/src/lib/static-model/static-model.ts:429]: To display on the alphabetic order of the select, without adding a function

Comment (phpgeekfr) [reply libs/utils/src/lib/static-model/static-model.ts:429]: but you don't use it in your PR

**Review by phpgeekfr — APPROVED** [review]
LGTM ;)

**Comment by RemcoSimonides** [comment]
@jeremie74 do you mind referring the issue number like this #6178 in the description of your PRs? That way I can easily find the PR from the issue, and the issue from the PR.

**Pull request merged**